### DISCUSSION TITLE
PGC内容请求失败【求助】

### DISCUSSION THREAD

**Discussion by CC-007** [Richasy/Bili.Uwp#1158] [问答]
漫游设置代理已经填写了，可哔哩还无法观看港澳台的番。

**Comment by Richasy** [comment]
参考 [简单地漫游一下](https://github.com/Richasy/Bili.Uwp/wiki/%E7%AE%80%E5%8D%95%E5%9C%B0%E6%BC%AB%E6%B8%B8%E4%B8%80%E4%B8%8B) 来配置自己的代理服务
